account_commission/models/account_move.py

```diff
@@ -49,31 +49,28 @@ def action_post(self):
         return super().action_post()
 
     def button_draft(self):
-        """Check settled lines and put settlements associated to the invoices in
-        invoice when state is changed from cancel to draft.
+        """Restrict reset to draft in invoice if there are settlement lines for invoice lines and 
+        put settlement state in invoiced when bill related with settlment is 
+        changed from cancel to draft again.
```

Comment:
This is not incorrect but it is a bit confusing because here two different functions are connected into one.  I'd personally put individual comments above relevant lines.